src/helpers.js

```diff
@@ -0,0 +1,169 @@
+function camelize(str) {
+  return str.replace(/\W+(.)/g, (match, chr) => chr.toUpperCase());
+}
+
+function uiColor(color) {
+  return `UIColor(red: ${color.r}/255, green: ${color.g}/255, blue: ${
```

Comment:
Why here are newlines here?

Reply:
Line too long and it's standard prettier auto formatting